pkg/kubemark/controller.go

```diff
@@ -207,35 +242,31 @@ func (kubemarkController *KubemarkController) addNodeToNodeGroup(nodeGroup strin
 }
 
 func (kubemarkController *KubemarkController) removeNodeFromNodeGroup(nodeGroup string, node string) error {
-	pods, err := kubemarkController.externalCluster.podLister.List(labels.Everything())
-	if err != nil {
-		return err
+	pod := kubemarkController.getPodByName(node)
+	if pod == nil {
+		return fmt.Errorf("can't delete node %s from nodegroup %s. Node does not exist", node, nodeGroup)
```

Comment:
As discussed offline, can we print a warning in this case and return nil? Returning an error results in stopping resize and failing a test if even one node was scaled down by Cluster Autoscaler in the meantime.